tests/pytests/functional/modules/test_cmdmod.py

```diff
@@ -518,17 +529,15 @@ def test_cmd_run_whoami(cmdmod, running_username):
     assert user.lower() == cmd.lower()
 
 
-@pytest.mark.skip_on_windows(
-    reason="Skip on Windows, Windows enviroment issues for this test - test needs fixing"
-)
-@pytest.mark.skip_on_linux
 @pytest.mark.slow_test
+@pytest.mark.skip_unless_on_windows(reason="Minion is not Windows")
 def test_windows_env_handling(cmdmod):
     """
     Ensure that nt.environ is used properly with cmd.run*
     """
-    out = cmdmod.run("set", env={"abc": "123", "ABC": "456"}).splitlines()
-    assert "abc=456" in out
+    out = cmdmod.run("cmd /c set", env={"abc": "123", "ABC": "456"}).splitlines()
+    assert "abc=123" in out
+    assert "ABC=456" in out
```

Comment:
when we did this on the command line in Windows, got abc=456, especially since Windows is case-insensitive , are these the correct asserts ?

Reply:
Yes, it was wrong before. It was checking for `abc=456` before... which was NOT was we were passing to the environment.